src/util/Constants.js

```diff
@@ -109,6 +109,18 @@ function makeImageUrl(root, { format = 'webp', size } = {}) {
   if (size && !AllowedImageSizes.includes(size)) throw new RangeError('IMAGE_SIZE', size);
   return `${root}.${format}${size ? `?size=${size}` : ''}`;
 }
+
+function makeStickerUrl(root, { format = 'png', size, passthrough = true } = {}) {
+  const params = new URLSearchParams();
+  if (size) {
+    if (!AllowedImageSizes.includes(size)) throw new RangeError('IMAGE_SIZE', size);
+    params.append('size', size);
+  }
+  params.append('passthrough', passthrough);
+  const stringifiedQuery = params.toString();
+  return `${root}.${format}${stringifiedQuery .length ? stringifiedQuery  : ''}`;
```

Comment:
Just to make ESLint happy:
```suggestion
  return `${root}.${format}${stringifiedQuery.length ? stringifiedQuery  : ''}`;
```